CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support device groups in device triggers.
 - Initial device introspection during device registration.
 
+### Fixed
+- Data trigger paths matching with interface mappings.
```

Comment:
- Fix a bug preventing triggers to be installed on parametric endpoints containing an underscore.